File: sdk/src/main/java/com/shiftpayments/link/sdk/api/vos/responses/card/SelectBalanceStore.java
package com.shiftpayments.link.sdk.api.vos.responses.card;

import com.google.gson.annotations.SerializedName;
import com.shiftpayments.link.sdk.api.vos.responses.workflow.AllowedBalanceType;

import java.util.ArrayList;

public class SelectBalanceStore {

    @SerializedName("allowed_balance_types")
    public ArrayList<AllowedBalanceType> allowedBalanceTypes;

    public SelectBalanceStore(ArrayList<AllowedBalanceType> allowedBalanceTypes) {
        this.allowedBalanceTypes = allowedBalanceTypes;
    }
}

File: mobile/src/main/java/com/aptopayments/mobile/data/card/SelectBalanceStore.kt
package com.aptopayments.mobile.data.card

import com.aptopayments.mobile.data.workflowaction.AllowedBalanceType
import java.io.Serializable

data class SelectBalanceStore(
    val allowedBalanceTypes: List<AllowedBalanceType>?
) : Serializable
